90anti-evil-maid/anti-evil-maid-unseal

```
@@ -58,6 +58,8 @@ fi
 log "Mounting $DEV (\"$LABEL\")..."
 mkdir -p "$MNT"
 mount -t ext4 -o ro "$DEV" "$MNT"
+# this way an error prior to unmounting will keep the device usable after initrd
+trap 'umount --quiet "$MNT" || true' EXIT
```

The `|| true` will hide other errors too, not only the "not mounted" one. Maybe clear the trap when unmounting explicitly later in this script instead?